app/assets/javascripts/backbone/views/log_modal_view.coffee

```diff
@@ -24,14 +23,27 @@ class MS.Views.LogModalView extends Backbone.View
   replaceContent: (html) ->
     @$el.html(html)
     new MS.Views.TranslationsView(el: @$('[data-content-translatable="project_log"]'))
+    @$el.find('.alert').hide()
     @$('.modal').modal('show')
     MS.loadingIndicator.hide()
 
   submitForm: (e) ->
     e.preventDefault()
-    @$('form').submit()
-    @$('.modal').modal('hide')
-    @submitted = true
+    $form = @$('form')
+    formIsEmpty = true
+
+    # Check to make sure summary is completed for at least one language
+    # Only submit form if summary is present
+    $form.find("[data-translatable='common.summary']").each ->
+      if ($.trim($(this).val()) != '')
+        $form.submit()
+        @submitted = true
+        formIsEmpty = false
```

Comment:
If there are multiple summaries then the form will get submitted multiple times.

You could take the `submit()` outside the loop and check if the @submitted flag is true when you get outside the loop.

Also you don't need both the `formIsEmpty` and `@submitted` flags, right? They are just opposites?